exercises/transpose/src/test/java/TransposeTest.java

```diff
@@ -0,0 +1,259 @@
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.Before;
+
+import static org.junit.Assert.assertEquals;
+
+public class TransposeTest {
+    private Transpose transpose;
+
+    @Before
+    public void setup() {
+        transpose = new Transpose();
+    }
+
+    @Test
+    public void emptyString() {
+        String input = "";
+        String expected = "";
+
+        assertEquals(expected, transpose.transpose(input));
+    }
+
+    @Test
+    @Ignore("Remove to run test")
+    public void twoCharsInARow() {
+        String input = "A1";
+        String expected = "A" +
+                "\n1";
+
+        assertEquals(expected, transpose.transpose(input));
+    }
+
+    @Test
+    @Ignore("Remove to run test")
+    public void twoCharsInAColumn() {
+        String input = "A\n" +
+                "1";
+        String expected = "A1";
+
+        assertEquals(expected, transpose.transpose(input));
+    }
+
+    @Test
+    @Ignore("Remove to run test")
+    public void simple() {
```

Comment:
I know you've got these names from the canonical data, so it's not your fault, but `simple` is not a very descriptive test name. Maybe `transposeSimpleMatrix` or `simpleMatrix`? While we should use the test data in the canonical data, we don't have to stick to the exact test names given in the description :)